core/priv/repo/migrations/20211018190850_add_crew.exs

```diff
@@ -0,0 +1,77 @@
+defmodule Core.Repo.Migrations.AddCrew do
+  use Ecto.Migration
+
+  def up do
+    create table(:crews) do
+      add(:reference_type, :string, null: false)
+      add(:reference_id, :string, null: false)
```

Comment:
Can this be an integer since most (all?) of the target types have an integer as their primary key?

Reply:
Sure!